apps/codesearch/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- dims
+# TODO: Anyone else??
```

Comment:
Can we address ownership before we merge this ? I would like to see adoption from a SIG.

Reply:
@BenTheElder I think you mentioned a few weeks ago you may be willing to help support ownership of codesearch. Do you possibly know if you could get support from a SIG to be an OWNER?